server/db/007-addbbatterymonitoring.sql

```diff
@@ -0,0 +1,21 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 7;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE registry ADD COLUMN button_serial_number text DEFAULT NULL;
+        ALTER TABLE registry ADD COLUMN button_battery_level text DEFAULT NULL;
```

Comment:
i think it makes more sense for battery level to be an integer? ie. from 0 to 100